Dockerfile

```diff
@@ -5,5 +5,6 @@ RUN apk add --no-cache git
 
 RUN mkdir /tmp/src
 ADD . /tmp/src
-RUN pip3 install --no-cache-dir /tmp/src
-
+RUN pip3 install --no-cache-dir /tmp/src && \
+    cp /tmp/src/docker/git-credential-env /usr/local/bin/git-credential-env && \
```

Comment:
Minor nit, but I think there's no advantage in this being the same RUN as the pip install. If so, we should use a new RUN for clarity.